gocd/gocd.go

```diff
@@ -101,12 +102,23 @@ func client() *http.Client {
 }
 
 // NewServerConfig Create a Server object from a config
-func NewServerConfig(host string, port string, user string, password string) *Server {
+func NewServerConfig(host string, port string, user string, password string, timeoutStr string) *Server {
+
+	// timeout casting to seconds
+	timeout := time.Duration(120 * time.Second)
+	i, err := strconv.Atoi(timeoutStr)
+	if err == nil {
+		timeout = time.Duration(i) * time.Second
+	} else {
+		log.Warn("Failed to convert timeout to seconds", err)
```

Comment:
@frank0631 Nitpicky, I know, but I think you need a space at the end of the string, right? Or else the err message will be concatenated directly after "seconds".  I would suggest `"seconds: ", err`